platform/linux-generic/odp_ipsec_sad.c

```diff
@@ -423,6 +423,42 @@ static uint32_t esp_block_len_to_mask(uint32_t block_len)
 	return block_len - 1;
 }
 
+/* Return supported power-of-two AR window size */
+static uint32_t ipsec_antireplay_ws(uint32_t win_size)
```

Comment:
This function (which does not work for all possible values of win_size) seems unnecessary as the ROUNDUP_POWER2_U32() macro of odp_align_internal.h could be used.

Reply:
We tried to avoid this function for rounding up window size to the next power of two, but couldn't find an ODP API that you suggested or similar to this, will check this API and try to avoid using this function.